### ISSUE TITLE
0.22 release (Spirited Giraffe)

### ISSUE THREAD

**Issue by wizzomafizzo** [wizzomafizzo/starcheat#178]
What needs to be done before next release. Only one thing broke, otherwise things seem stable:
- [x] Random name generation has changed, crashes
- [x] Disable quests button if metadata not exist
- [x] Find stray logging filling up debug logs
- [x] Check over metadata and player for any new parts
- [x] Disable quests button if quests don't exist (brand new player)

**Comment by wizzomafizzo** [comment]
Name generation uses markov chains now with a source I can't find. Going to disable it entirely for now and open another ticket to consider it. Not really worth the work to implement as far as I'm concerned.

**Comment by wizzomafizzo** [comment]
@chrmoritz when you have some time free could you please check I haven't screwed anything up. I did the brew file as well

**Comment by chrmoritz** [comment]
lgtm, except that no appveyor build was triggered (did it manually and we have to upload the artifacts by ourself to github release when they are ready) and I added a revision to the git tag in the homebrew formula to fix audit warnings in recent homebrew versions

**Comment by chrmoritz** [comment]
uploaded the windows build and released version 0.22

**Comment by wizzomafizzo** [comment]
ahh ok cool i'll remember to update that next time too. thanks you rock ^_^
